resources/views/vendor/mail/html/header.blade.php

```diff
@@ -0,0 +1,17 @@
+<tr>
+<td class="header" style="vertical-align:middle">
+<a href="{{ $url }}" style="display: inline-block;">
+@switch (trim($slot))
+@case('Laravel')
+<img src="https://laravel.com/img/notification-logo.png" class="logo" alt="Laravel Logo">
+@break
+@case('Urán')
+<img src="https://uran.eotvos.elte.hu/img/uran_blue.png" width="60" alt="Urán Logó">
```

Comment:
Will work after the new release

Reply:
You can get these using the asset() function.

Reply:
Even better, you could embed them: https://laravel.com/docs/7.x/mail#inline-attachments

Reply:
Done